doc/reference_accessor.xml

```diff
@@ -1184,10 +1568,7 @@ postgis=# SELECT ST_IsClosed('MULTIPOINT((0 0), (1 1))'::geometry);
 	<refentry id="ST_IsCollection">
 	  <refnamediv>
 		<refname>ST_IsCollection</refname>
-
-                <refpurpose>Returns <varname>TRUE</varname> if the argument is a
-                  collection (<varname>MULTI*</varname>, <varname>GEOMETRYCOLLECTION</varname>, ...)
-                </refpurpose>
+      <refpurpose>Tests if a geometry is a geometry collection type.</refpurpose>
```

Comment:
this change lost an important note that MULTI* is also a collection according to this.

Reply:
The detail is described in the function page.  Should the comment be made longer/more descriptive as well?  In general I've been trying to make the summary comments more concise.

And on a related note, there should be a better section describing ALL the geometry types supported by PostGIS, which can be linked to.  This is on my todo list...